lunar/release-stretch-build.yaml

```diff
@@ -12,6 +12,8 @@ notifications:
   - mikael+buildfarm@osrfoundation.org
   maintainers: true
 package_blacklist:
+package_whitelist:
+  - catkin
```

Comment:
ok thanks, fixed in https://github.com/ros-infrastructure/ros_buildfarm_config/pull/75/commits/4449d73faded10959c1ad872c7bbfe313124de13 for all files using black/whitelists